aws/main.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_iam_openid_connect_provider" "stacks_openid_provider" {
 
 resource "aws_iam_role" "stacks_role" {
   name               = "stacks-${var.tfc_organization}-${var.tfc_project}-${var.tfc_stack}"
-  assume_role_policy = data.aws_iam_policty_document.stacks_role_policy.json
+  assume_role_policy = data.aws_iam_policy_document.stacks_role_policy.json
```

Comment:
Fixed a typo here. Can be ignored if N/A.

Reply:
Nice, I fixed this here already: https://github.com/liamcervante/terraform-stacks-identity-tokens/commit/1b24d1c3512b1a23d1b5e6732830837c8994a852. It's not identifying a conflict though, so we can just merge this as well 😅